src/fragments/lib/auth/js/emailpassword.mdx

```diff
@@ -288,9 +288,55 @@ Based on the two parameters returned in the user object, you can then call the c
 
 If the user has SMS Multi-Factor Authentication (MFA) enabled, Cognito responds with a challenge that requires the user to provide an SMS verification code. This is where you can trigger `Auth.confirmSignIn()` and pass in the code provided by the user.  Amplify will then verify with Cognito that the SMS code is valid and complete the sign in process via Cognito returning access, ID, and refresh tokens that are all handled by Amplify internally.
 
-```javascript
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
 import { Auth } from 'aws-amplify';
 
+type ConfirmSignInWithMFAParameters = {
```

Comment:
these `type` name should be generic to handle any `signIn` call. I don't think a project would have different `sign-in` calls all over the code base to handle different purposes.